plugins/pipelines/remove.js

```diff
@@ -52,6 +55,18 @@ module.exports = () => ({
                                 + 'does not have admin permission for this repo');
                         }
                     })
+                    .catch((error) => {
+                        // Lookup whether user is admin
+                        const adminDetails = request.server.plugins.pipelines
```

Comment:
I think you can reuse the banners code like this instead: https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/builds/update.js#L67
Not sure what makes more sense

Reply:
Makes sense! Better than duplicating it.